@navikt/ds-react/copy-to-clipboard/src/CopyToClipboard.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import { forwardRef, HTMLAttributes, useEffect, useRef, useState } from "react";
+import { Files } from "@navikt/ds-icons";
+import "@navikt/ds-css/baseline/utility.css";
```

Comment:
Blir css-en i `utility.css` duplisert per komponent som importerer den på denne måten? Og avhengigheten til `.sr-only`-regelen blir temmelig implisitt her, noe som gjør det vanskelig å refactore i framtiden. Er det noen bedre måter å gjøre dette på? Kanskje vi kan lage en React-komponent for `sr-only`?